graphite_metrictank.py

```diff
@@ -7,7 +7,7 @@
 from flask import g
 import structlog
 logger = structlog.get_logger('graphite_api')
-import json
+import simplejson as json
```

Comment:
It is supposed to be much faster at decoding.  But haven't seen it make a difference.